contrib/go-chi/chi.v5/chi.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package chi provides tracing functions for tracing the go-chi/chi package (https://github.com/go-chi/chi).
```

Comment:
You should update the documentation too. Please search the entire package for `go-chi` and update the paths wherever it makes sense, including docs.